src/content/team/team.json

```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Daniel Wang",
+      "team": {
+        "name": "Eboard",
+        "role": "Head of Boolin"
```

Comment:
not a real role

Reply:
+1

Reply:
+10000

Reply:
Bro called the branch "patch-1"😭

Reply:
bro blame github - they autonamed the branch for me lmao